### ISSUE TITLE
Expression will out of momery when sicp package installed on racket-cs

### ISSUE THREAD

**Issue by mofeng-me** [sicp-lang/sicp#32]
* My OS Version is MacOS Catilina 10.15.1
* Racket-CS version is 7.5
* Details
When I installed sicp package on dr-racket-cs ,the express will show error "out of memory".The code is
```
#lang sicp
 (/ (+ 5 
       4 
       (- 2 (- 3 (+ 6 (/ 4 5))))) 
    (* 3 
       (- 6 2) 
       (- 2 7))) 
```
And also I tried other exercises in sicp.
but I will work well when I use `#lang racket`.

When I uninstalled `dr-racket-cs` then installed `drracket` and reinstalled sicp packge ,it will work well!

**Comment by soegaard** [comment]
Thanks for reporting this bug. Turns out it is due a bug in Racket CS. 
I'll leave this issue open until it is fixed.

**Comment by sorawee** [comment]
RacketCS was just fixed. Tomorrow, https://www.cs.utah.edu/plt/snapshots/ should have a new RacketCS binary. You can try install it and see if the issue is fixed or not.

**Comment by mofeng-me** [comment]
@soegaard Yes, the issue is fixed with "Snapshot: 20191126-70459f4105"

**Comment by sorawee** [comment]
Cool, I will close the issue then
